ts.js

```diff
@@ -0,0 +1,221 @@
+(function () {
+   var commonRules = require('./common-rules');
+   var es6Rules = require('./es6-rules');
+   var typescriptRules = {
+      '@typescript-eslint/adjacent-overload-signatures': 'error',
+      '@typescript-eslint/array-type': [
+         'error',
+         {
+            'default': 'array-simple'
+         }
+      ],
+      '@typescript-eslint/ban-types': [
+         'error',
+         {
+            'types': {
+               'Object': {
+                  'message': 'Avoid using the `Object` type. Did you mean `object`?'
+               },
+               'Function': {
+                  'message': 'Avoid using the `Function` type. Prefer a specific function type, like `() => void`.'
+               },
+               'Boolean': {
+                  'message': 'Avoid using the `Boolean` type. Did you mean `boolean`?'
+               },
+               'Number': {
+                  'message': 'Avoid using the `Number` type. Did you mean `number`?'
+               },
+               'String': {
+                  'message': 'Avoid using the `String` type. Did you mean `string`?'
+               },
+               'Symbol': {
+                  'message': 'Avoid using the `Symbol` type. Did you mean `symbol`?'
+               }
+            }
+         }
+      ],
+      '@typescript-eslint/consistent-type-assertions': 'error',
+      '@typescript-eslint/consistent-type-definitions': 'error',
+      '@typescript-eslint/dot-notation': 'error',
+      '@typescript-eslint/explicit-member-accessibility': [
+         'error',
+         {
+            'accessibility': 'no-public'
+         }
+      ],
+      'indent': 'off',
+      '@typescript-eslint/indent': [
```

Comment:
so some of these basic formatting ones do translate?

Reply:
yeah they all translated fine, its just not the preferred way of doing things since there are better tools now